spec_creation/final_sfbayarea/train_bus_ebike_mtv_ucb.json

```diff
@@ -603,22 +588,9 @@
-                      },{
-                        "relation_id": 2718359,
-                        "start_node": 6410561636,
-                        "end_node": 6383982769,
-                        "valid_start_fmt_date": "2019-09-01",
-                        "valid_end_fmt_date": "2020-04-30",
-                        "polyline": "_|{eFvtiiVBL??Fh@DX@R??DX??BTDf@??b@`D??l@zETdB??VvB??l@`FR`B??TdB??j@zEL`A??\vC??LbAPnA??Fn@??DTFf@??Hj@J^NZ??F^Hd@Fb@n@jFFf@Bf@Af@Ed@Kd@Qd@S^??OTSPSLUJWH??W@UA]@k@B??I?y@@K@??Q@??cDD_@@[@[B[BaAL??_@H[HYHs@TUFYDUDUB??UBSD??[C[CYGWGWMWM??CASM??]OYM??ICEAWG??UCUCWAU?oAF??eCJ??yAF??m@B??kAH??gAH[@u@DM@[BSD??E?_@HODM@??sAV??LpB@R??@R?Z??ARAh@?ZDX??z@S??XE??`AS??J`A??LdAFb@NxAB^??T|B??Ad@ENGJMHk@L??YFi@NUHMDc@Ta@Z??Y@YFSEGIEQ?SBMHMd@Y??tEu@??jDaA??f@K??bCu@l@Wt@Y??dN{Ep@W??jGeBlASfCUjAIhBCR?n@?`AHfAX|@^`Al@n@p@n@v@j@`A^`A`@jA\`BdA~F??d@rCh@tD??`AlJt@rHXnC??j@vFF~@BdAA`ACbAI~AQpCChA@hAB`AJbAf@rDTpA??BZ??Fn@Ft@Dr@Bt@?t@A`AA|A?hA?~@Bz@Fv@Fx@J`AJ~@d@nDRfAdA`GHd@XtA\nA^hAfBzE??b@jBR|@l@nCVfBRrBRvBP`CL`Ct@fU??LbDJvCLrCLjCNbCP|CPhCRnCxMteBJfALfALfAPbAP`ANz@Pv@Pt@Pr@Vx@Tt@Rp@Vp@Xt@Xt@Zt@Zp@pRda@??Vf@Vb@Xd@Xb@Z`@V\^d@^d@^`@\\\Z\ZtFxE|AfB??vChD??rFdG??hDvD??nyB`bCxKrLv@~@f@v@??|AbDH^JXh@jBFVDZBX??Jz@@z@Cl@E\GXKd@_@`A??_@z@??gAtA??iAxAUZmA`BKL??y@dA??Wl@OZK^I^I`@Qt@IXMT??GJ??k@v@MSGI??[c@??Y_@??[c@??]e@??]e@??[c@??]c@??[e@??W]??W]??W]??W]??g@q@??[e@??]c@??]e@??W]??Y_@j@w@`@j@??X`@??VZ"
-                      }
+                    "polyline": [
+                      "_|{eFvtiiVBL??Fh@DX@R??DX??BTDf@??b@`D??l@zETdB??VvB??l@`FR`B??TdB??j@zEL`A??\\\\vC??LbAPnA??Fn@??DTFf@??Hj@J^NZ??F^Hd@Fb@n@jFFf@Bf@Af@Ed@Kd@Qd@S^??OTSPSLUJWH??W@UA]@k@B??I?y@@K@??Q@??cDD_@@[@[B[BaAL??_@H[HYHs@TUFYDUDUB??UBSD??[C[CYGWGWMWM??CASM??]OYM??ICEAWG??UCUCWAU?oAF??eCJ??yAF??m@B??kAH??gAH[@u@DM@[BSD??E?_@HODM@??sAV??LpB@R??@R?Z??ARAh@?ZDX??z@S??XE??`AS??J`A??LdAFb@NxAB^??T|B??Ad@ENGJMHk@L??YFi@NUHMDc@Ta@Z??Y@YFSEGIEQ?SBMHMd@Y??tEu@??jDaA??f@K??bCu@l@Wt@Y??dN{Ep@W??jGeBlASfCUjAIhBCR?n@?`AHfAX|@^`Al@n@p@n@v@j@`A^`A`@jA\\\\`BdA~F??d@rCh@tD??`AlJt@rHXnC??j@vFF~@BdAA`ACbAI~AQpCChA@hAB`AJbAf@rDTpA??BZ??Fn@Ft@Dr@Bt@?t@A`AA|A?hA?~@Bz@Fv@Fx@J`AJ~@d@nDRfAnAfHXtA\\\\nA^hAfBzE??b@jBR|@l@nCVfBRrBRvBP`CL`Ct@fU??LbDJvCLrCLjCNbCP|CPhCRnCxMteBJfALfALfAPbAP`ANz@Pv@Pt@Pr@Vx@Tt@Rp@Vp@Xt@Xt@Zt@Zp@pRda@??Vf@Vb@Xd@Xb@Z`@V\\\\^d@^d@^`@\\\\\\\\\\\\Z\\\\ZtFxE|AfB??vChD??rFdG??pDjD??lyB~aCxKtLv@|@h@x@??x@dB??h@p@JJJPRd@R`@R`@x@`BNVFJFHFDFFHBHBH?H?JCJCFGHIFKDMDO?O?QAMCMEMGKGGmCyC??KMIIIGIGKGMCKCMAKAK@M?KBODKDSJKDIDKBK@w@N??SDIDIBGFKH??SVMP??OROT??w@`A??kA~A??uAdB??i@u@??mAcBMQ??OPs@`AUZ??MQOU??Q?GK??W[??a@i@??W\\\\??ED??SX??KJ??KLGH??OR??CDAH?H@H@B??LV??DHBB??PT",
```

Comment:
here's where we would want to add start/end tags that we would use for the polyline list.

Reply:
```
[
  {
     "polyline": "......"
     "start_fmt_date": ...
     "end_fmt_date": ...
   },
  {
     "polyline": "......"
     "start_fmt_date": ...
     "end_fmt_date": ...
  }
]
```